scipy/linalg/flinalg.py

```diff
@@ -14,7 +14,7 @@ def __getattr__(name):
     if name not in __all__:
         raise AttributeError(
             "scipy.linalg.flinalg is deprecated and has no attribute "
-            f"{name}. Try looking in scipy.linalg instead.")
+            f"{name}.")
```

Comment:
Wasn't sure about the best course of action here. Everything in the `__all__` of this module doesn't appear in `scipy.linalg`. This means if `_sub_module_deprecation` was used then they would all be stated for removal from `__all__` in 1.13.0 but then there would be nothing left in the module. However, the module isn't due to be removed until 2.0.0 hence why I just modified the message directly.

Reply:
I find it strange that we're keeping around an empty module, but the warning has already promised that'd we'd only remove it for 2.0.0, so 🤷

Reply:
I guess nobody anticipated that we would touch Fortran code. But I think that promise is void now.

Reply:
See https://github.com/scipy/scipy/issues/19072